docs/design/.pages

```diff
@@ -1,5 +1,6 @@
 nav:
   - index.md
+  - autoware-architecture
   - autoware-concepts
```

Comment:
https://github.com/autowarefoundation/autoware-documentation/pull/86#discussion_r854916501

```suggestion
  - autoware-concepts
  - autoware-architecture
```

Reply:
Done in https://github.com/autowarefoundation/autoware-documentation/commit/b9a3f80b26b99714f8b16cb16038677b970bc1ec